qgsquick/from_qgis/plugin/components/qgsquickcheckboxcomponent.qml

```diff
@@ -0,0 +1,90 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.8
+import QtQuick.Templates 2.1 as T
+import "../" // import styling singleton
+
+T.CheckBox {
+    id: control
+
+    signal checkboxClicked( var buttonState )
+
+    property var spaceWidth: height / 4.0
+    property var baseSize: height / 5.5
+    property var baseColor: "black"
```

Comment:
PR introduces new styling object for the checkbox widget, which is good. However, its not used, only defined in `qgsquickfeatureformstyling.qml` and redefined for Input in `featurepanel.qml`. Additionally I think import for styling singleton is not needed.

Reply:
Rather use something like `customStyle.checkbox.baseColor` instead of "black".

Reply:
Thanks for review, I added manual tests and changed the UI things.